test/uk/gov/hmrc/tai/repositories/EmploymentRepositorySpec.scala

```diff
@@ -1356,7 +1371,7 @@ class EmploymentRepositorySpec extends BaseSpec {
           npsConnector = mockNpsConnector,
           employmentBuilder = mockEmploymentBuilder)
 
-        val result = controller.employment(nino, 2).futureValue
+        val result = controller.employment(nino, 1).futureValue
```

Comment:
Why has this changed?

Reply:
interanally it uses below
employmentById(id: Int): Option[Employment] = employments.find(_.sequenceNumber == id). So to match provided id with sequence number I have changed it to 1.